cordova-lib.js

```diff
@@ -20,12 +20,6 @@
 var common = require('cordova-common');
 
 exports = module.exports = {
-    set binname (name) {
-        this.cordova.binname = name;
-    },
-    get binname () {
-        return this.cordova.binname;
-    },
```

Comment:
`binname` attribute is used by cordova-cli, hope we can remove it in the future - my bad again

Reply:
We can't. This is used by other tools that wrap cordova-lib, like `ionic`, to display their command-line tool name in help and error messages instead of `cordova`